### PR TITLE
README page update (license)

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

Add more information about licensing in the README page.

# Checklist

- [n.a.] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [n.a. ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [ n.a.] Documentation module page added or updated.
- [ n.a.] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [ n.a.] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ n.a.] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +48,16 @@ Minimum system requirements: C++11 enabled compiler, [cmake](http://cmake.org),
 
 DGtal can be compiled on Microsoft Windows system using Visual Studio 2014 (or newer): Generate the Visual Studio project using windows [cmake](http://cmake.org) tool and compile the DGtal solution (you may also need to set the cmake variable ```BUILD_SHARED_LIBS``` to false).
 
+
+License
+=======
+
+DGtal core libarary is under the GNU LGPL v3 license (see [LICENSE.txt](LICENSE.txt)). DGtal documentation and website materials are under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 ](http://creativecommons.org/licenses/by-nc-sa/4.0/).
```
Comment (kerautret) [inline README.md]: the looks not working (due to the .txt )

**Comment by dcoeurjo** [comment]
you're right;) thx

File: README.md [hunk 2]
```diff
@@ -47,6 +48,16 @@ Minimum system requirements: C++11 enabled compiler, [cmake](http://cmake.org),
...
+
+If some dependencies are activated, resulting binaries may follow the dependencies licenses. More precisely:
+* If the library is built with the 3D internal viewer using [libQGLViewer](http://libqglviewer.com) (```WITH_QGLVIEWER``` flag set to ```true```), then resulting binaries libQGLViewer license (GNU GPL or commercial, see [libQGLViewer licensing](http://libqglviewer.com/dualLicensing.html)).
```
Comment (kerautret) [inline README.md]: it looks something is missing at the end:
=> "then the resulting binaries are under the libQGLViewer license (GNU...)"?

**Comment by kerautret** [comment]
fine merging

**Pull request merged**